src/lib/es2015.collection.d.ts

```diff
@@ -30,7 +30,7 @@ interface Map<K, V> {
 interface MapConstructor {
     new (): Map<any, any>;
     new <K, V>(entries?: readonly (readonly [K, V])[] | null): Map<K, V>;
-    readonly prototype: Map<any, any>;
+    prototype: Map<any, any>;

```

Comment:
I'm concerned with this change as `prototype` is actually read-only on `Map`, `Set`, `WeakMap`, `WeakSet`, etc.:

![image](https://github.com/user-attachments/assets/8be82e69-b910-42b7-b6c3-7cea030c2ec0)